docs/valuesYaml-valuesFiles.md

```diff
@@ -0,0 +1,64 @@
+# Understanding Helm values.yaml vs. Fleet valuesFiles
+
+When using Fleet with Helm, you often confuse the chart’s built-in `values.yaml` with additional values files referenced in `fleet.yaml`. These files serve different purposes, and it’s important to understand how they interact.
+
+![Understanding Helm values.yaml vs Fleet valuesFiles with best practices](../static/img/helm-value-fleet-yaml.svg)
+
+## Helm chart’s values.yaml
+
+Every Helm chart includes a `values.yaml`. This file contains the default configuration values used when deploying the chart.
+
+* It’s automatically applied by Helm and Fleet. You don’t need to reference it in `fleet.yaml`.  
+* It always acts as the baseline when the Fleet agent installs the chart.  
+* Any overrides you provide later (via Fleet `valuesFiles` or `values: parameters`) replace or extend these defaults.
```

Comment:
Wouldn't it make sense to also mention [valuesFrom](https://fleet.rancher.io/gitrepo-content#using-valuesfrom) here?